### PR TITLE
Update radioplayer plugin to use new API and rpuid.

### PR DESCRIPTION
**Pull request by mstratford**
From https://support.radioplayer.co.uk/support/solutions/articles/48001224489-upgrading-and-using-new-radioplayer-ingest-apis

Needs testing when we have our new auth credentials.

### CODE REVIEW

File: blast_plugins/radioplayer.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def poke_track(self, now_playing: Optional[NowPlaying]):
         r = requests.post(
           url,
           auth = auth,
-          data = data
+          params = data
```
Comment (markspolakovs) [inline blast_plugins/radioplayer.py:83]: Why this change?

**Review by markspolakovs — APPROVED** [review]
Code LGTM with one minor comment, would be good to get it tested :)

**Comment by mstratford** [comment]
Working with new creds from radioplayer.

File: blast_plugins/radioplayer.py (hunk 1 above)
Comment (mstratford) [reply blast_plugins/radioplayer.py:83]: It seems params uses URL parameters, actually matching the cURL example above in the code. It seems the new API doesn't accept the data as POST data, only params, so was giving a 500 error.

**Comment by mstratford** [comment]
Screenshot_20230309_225042_Radioplayer
